Dockerfile

```diff
@@ -0,0 +1,39 @@
+# The builder image, used to build the virtual environment
+FROM python:3.11-slim-buster as builder
```

Comment:
python3.9 is the minimum required in poetry, so which one is it?
(You mentioned to me python3.12 being mandatory for some typing-related stuff)